buildSrc/subprojects/performance-testing/src/main/groovy/gradlebuild.performance-templates.gradle

```diff
@@ -52,18 +52,20 @@ performanceTest.registerTestProject("withVerboseTestNG", JvmProjectGeneratorTask
     projects = 1
     sourceFiles = 10
     subProjectTemplates << 'with-verbose-testng'
+    daemonMemory = '256m'
```

Comment:
Would it make sense to extract these to constants, like `SMALL_MEMORY`, `LARGE_MEMORY` etc.?

Reply:
I don't think that makes sense. The memory requirements would not necessarily change for all projects which use the same constant at the same time. The memory is a per project setting and accidentally some project use the same value.